CHAID/__main__.py

```diff
@@ -25,7 +26,16 @@ def main():
         config['alpha_merge'] = nspace.alpha_merge
     if nspace.min_samples:
         config['min_sample'] = nspace.min_samples
-    CHAID.from_pandas_df(df, nspace.independent_variables, nspace.dependent_variable[0], **config).print_tree()
+    tree = CHAID.from_pandas_df(df, nspace.independent_variables, nspace.dependent_variable[0], **config)
+
+    if nspace.classify:
+        predictions = pd.Series(tree.predict())
+        predictions.name = 'node_id'
+        df = pd.concat([df, predictions], axis=1)
+        print(df.to_csv())
```

Comment:
Not to terminal. `to_csv` takes a file name argument if we want to go that route. The brackets are ugly but needed for python 3 compatibility